alot/settings/manager.py

```diff
@@ -181,28 +179,26 @@ def _parse_accounts(config):
                 accounts.append(newacc)
         return accounts
 
-    def _process_xdg_setting(self, setting_name, fallback):
+    def _process_xdg_setting(self, setting_name):
         """
         Processes setting with $XDG_CONFIG_HOME as default value.
 
-        If config value starts with $XDG_CONFIG_HOME, expand it or replace
-        the whole setting with the fallback value.
+        If config value starts with $XDG_CONFIG_HOME, expand it. If the
+        environment variable is empty or not set, expand it to `~/.config`.
         Expand home directory (~) and update the path in _config object.
 
         :param setting_name: name of setting to be processed
         :type setting_name: str
-        :param fallback: value, if $XDG_CONFIG_HOME is empty or does not exist
-        :type fallback: str
         :returns: path
         :rvalue: str
         """
         path = self._config.get(setting_name)
         if path.startswith('$XDG_CONFIG_HOME'):
             if os.environ.get('XDG_CONFIG_HOME', False):
-                path = path.replace('$XDG_CONFIG_HOME',
-                                    os.environ['XDG_CONFIG_HOME'])
+                xdg_expanded = os.environ['XDG_CONFIG_HOME']
             else:
-                path = fallback
+                xdg_expanded = '~/.config'
```

Comment:
Instead of the if else stuff you can simply do `xdg_expanded = os.environ.get('XDG_CONFIG_HOME', '~/.config')`.

Reply:
Turns out that unfortunately that doesn't work, because if `XDG_CONFIG_HOME` is empty, it replaces `$XDG_CONFIG_HOME` with an empty string. The tests caught this 👍